homeassistant/components/comfoconnect/__init__.py

```diff
@@ -81,6 +81,7 @@ def _shutdown(_event):
 
     # Load platforms
     discovery.load_platform(hass, "fan", DOMAIN, {}, config)
+    discovery.load_platform(hass, "climate", DOMAIN, {}, config)
```

Comment:
Why is fan removed and now only the climate platform is set up?

Reply:
@springstan I checked this code and fan works correctly. HA set up both platforms here.

Reply:
Sorry I read that wrong. Best practice is to create a list constant `PLATFORMS` that contains all available platforms and to iterate over them and calling load_platform for each one.